src/scene/materials/standard-material.js

```diff
@@ -1088,7 +1093,9 @@ function _defineMaterialProps() {
     _defineFloat('reflectivity', 1);
     _defineFloat('occludeSpecularIntensity', 1);
     _defineFloat('refraction', 0);
-    _defineFloat('refractionIndex', 1.0 / 1.5); // approx. (air ior / glass ior)
+    _defineFloat('refractionIndex', 1.5); // approx. (air ior / glass ior)
```

Comment:
Ah, this one might be an issue in the future, just like how specular introduced a bunch of problems.